kerl

```diff
@@ -724,194 +693,6 @@ show_logfile() {
     l=t stderr "Please see $2 for full details."
 }
 
-maybe_patch() {
```

Comment:
I think we should keep this function as a no-op because I feel certain we will run into a situation in the future where we need it due to some kind of shenanigans either at the Linux compiler level or upstream. (No shade intended for the OTP team, but mistakes get made and released sometimes.)

Reply:
Sure. I removed it since we could always recover it from source control, but I can put it back.

Reply:
And is it just the function? No content? Or do we also restore the caller site?

Reply:
Yeah restore the call site too. I think passing in the params as-is is fine; it should be a no-op so won't hurt anything to supply them.

Reply:
8fbccc0

Reply:
Lemme check where this is called, then...

Reply:
8fbccc0